deepspeed/env_report.py

```diff
@@ -82,28 +83,46 @@ def debug_report():
     hip_version = None
     if hasattr(torch.version, 'hip'):
         hip_version = torch.version.hip
+    if get_accelerator().device_name() == 'cuda':
+        report = [
+            ("torch install path",
```

Comment:
Is it possible to simplify this list construction by refactoring out the common details related to torch and deepspeed? So that only the accelerator specific details are conditionally defined.

Reply:
Will lift common part and condition on accelerator specific code only.